contract/src/contract.js

```diff
@@ -26,8 +26,8 @@ const start = async (zcf) => {
   // can be accessed synchronously.
   const { amountMath, issuer } = zcfMint.getIssuerRecord();
 
-  const mintPayment = (extent) => (seat) => {
-    const amount = amountMath.make(extent);
+  const mintPayment = (seat) => {
+    const amount = amountMath.make(1000);
```

Comment:
Is there a reason the amount is inside the function rather than outside?  That's fine for my purpose, I'm just curious.

Reply:
It seemed easier to understand. It's creating new objects so not as efficient but...